doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
+A backend allows a file to be interpreted as keys in the key database such that any edits to the keys are reflected in the file and vice versa.  
+Additionally, the user can use this command to list the currently mounted backends by running the command with no arguments.  
+
+## USAGE
+
+`kdb mount [<path> <mountpoint>] [<plugin> [<config>] [..]]`  
```

Comment:
> I am very confused about the /sw/... keys?

These are normal cascading keys.

> Where are they stored in KDB?

Could be in any namespace or even somewhere else (if spec/sw/kdb/current/plugins specifies override or fallback)

> key not found

The message says it all ;) You need to create one if you want to change the default. But please describe this in the man page.